server/views/partials/searchForBookingHeader.njk

```diff
@@ -0,0 +1,64 @@
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+<div class="govuk-grid-column-full">
+    <div class="block-background">
+        <div class="govuk-!-width-one-half">
+            <form action="/search/booking" method="POST" novalidate>
+                <input type="hidden" name="_csrf" value="{{ csrfToken }}">
+
+                <label class="govuk-label bapv-booking-search-label" for="searchBlock1">
+                Enter the booking reference
+                </label>
+
+                <div class="bapv-booking-search">
+                {{ govukInput({
+                    classes: "govuk-input--width-2",
+                    id: "searchBlock1",
+                    name: "searchBlock1",
+                    spellcheck: false,
+                    value: searchBlock1,
```

Comment:
Where is `searchBlock1` coming from 🤔? (might have missed something or perhaps this is coming next...!)

Reply:
There are 4 search blocks, I haven't got as far as doing that bit yet 😁